test/src/specs/sync/ibd_process.rs

```diff
@@ -147,7 +147,16 @@ impl Spec for IBDProcessWithWhiteList {
             panic!("node6 can't generate blocks to 2");
         }
 
-        node0.connect_uncheck(node6);
+        let is_disconnect_already = wait_until(10, || {
```

Comment:
Why this checking matter? Better adds some comments to explain the reason.

Reply:
Sorry that I still don't understand. What if they don't disconnect each other?

Reply:
If it is not possible to confirm that both parties have been disconnected, Node0 and node6 will not synchronize during the test time, because node0 has confirmed that node6 is in the ibd state.